src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OpprettOppfølgingsOppgaveTask.kt

```diff
@@ -25,14 +27,20 @@ class OpprettOppfølgingsOppgaveTask(private val oppgaveService: OppgaveService,
     private val logger = LoggerFactory.getLogger(javaClass)
 
     override fun doTask(task: Task) {
```

Comment:
Synes denne metoden kan forbedres nå som vi har tatt inn barnetilsyn. Vi kan for eksempel ta i bruk sealed-class funksjonaliteten her, er åpen for at ikke alle synes det er bedre, men her er hvertfall mitt forslag.

All logikk som går under `skalOppretteVurderHenvendelseOppgave` kan ha egen funksjon i IverksettOvergangsstønad og IverksettBarnetilsyn (og som abstract fun i Iverksett-klasse). Hvor metoden som ligger i barnetilsyn-klassen bare returnerer false, og hvor all logikk som eksisterer fra før i oppgaveservice kan legges i IverksettOvergangsstønad.

Tasken kan hete OpprettOppfølgingsoppgaveForOvergangsstønadTask.

Dersom sealed class ikke ønskes, tenker jeg hvertfall at sjekken på stønadtype som er lagt til her kan legges til i `skalOppretteVurderHenvendelseOppgave`